kv/dist_sender.go

```diff
@@ -444,7 +441,7 @@ func (ds *DistSender) sendAttempt(trace *tracer.Trace, ba proto.BatchRequest, de
 	}
 
 	// TODO(tschottdorf) &ba -> ba
```

Comment:
Whoops. Definitely got it now.

On Sun, Sep 27, 2015 at 2:40 PM, Tamir Duberstein <notifications@github.com>
wrote:

> In kv/dist_sender.go
> <https://github.com/cockroachdb/cockroach/pull/2678#discussion_r40507673>:
>
> > @@ -444,7 +441,7 @@ func (ds *DistSender) sendAttempt(trace *tracer.Trace, ba proto.BatchRequest, de
> >  	}
> >
> >  	// TODO(tschottdorf) &ba -> ba
>
> this is still here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2678/files#r40507673>.
>